elastic4s-core/src/main/scala/com/sksamuel/elastic4s/requests/indexes/MappingHandlers.scala

```diff
@@ -28,12 +29,34 @@ trait MappingHandlers {
       }
     }
 
+    private def getMappings(mappings: Map[String, Any]) = {
+      mappings
+        .getOrElse("properties", getFieldMappings(mappings))
```

Comment:
This is the tricky bit: if the response doesn't have "properties" then it's either 

1. There are no mappings
2. There are mappings and it is a field mapping request (aka when we ask for specific fields in a get mapping request).